chunking_methods/gen_csv_labels.py

```diff
@@ -0,0 +1,177 @@
+"""Generates binary annotations using TweetyNet from weakly labeled audio.
+This file should be run from inside the PyHa directory. It also requires
+WTS_chunking.py to be added to the PyHa directory. 
+Input:     A path to a folder with audio files
+Output:    A csv with chunked, strongly-labeled annotations
+"""
+
+import os
+import sys
+from math import ceil
+import pandas as pd
+from pydub import AudioSegment
+# pylint: disable=import-error #this file gets put into PyHa
+from PyHa.IsoAutio import generate_automated_labels
+from WTS_chunking import dynamic_yan_chunking
+
+ISOLATION_PARAMETERS = {
+    "model" : "tweetynet",
+    "tweety_output": True,
+    "verbose" : True
+}
+
+CHUNK_DURATION = 5
```

Comment:
create config later?

Reply:
started doing this, but realized I might need to rewrite a bit of it so these functions can be called perhaps from other files. everything is basically a global rn and that's not ideal

Reply:
rewrite done!